roles/livekit/defaults/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+livekit_version: "v1.7"
+livekit_jwt_service_version: "sha-4a29504"
+livekit_user: "livekit"
+livekit_container_image_reference: "livekit/livekit-server:{{ livekit_version }}"
+livekit_config_path: "/opt/livekit"
+livekit_config_file: "livekit.yaml"
+livekit_container_config: "/etc/livekit.yaml"
+livekit_container_command: "--config {{ livekit_container_config }}"
+livekit_config_basic:
+  port: 7880
```

Comment:
As mentioned in the local PR, ideally external ports should be made configurable (at least if it doesn't complicate the configuration too much).